src/controllers/notifications.controller.js

```diff
@@ -0,0 +1,36 @@
+import NotificationService from '../services/notifications.service';
+import sendResult from '../utils/sendResult';
+
+const NotificationController = {
+  async getUserNotifications(req, res) {
+    const { userId } = req.userData;
+    return sendResult(
+      res,
+      200,
+      'Notifications',
+      await NotificationService.getAllNotifications(userId)
+    );
+  },
+
+  async getSingleNotification(req, res) {
+    return sendResult(
+      res,
+      200,
+      'Notification',
+      req.notification,
+    );
+  },
+
+  async updateNotificationStatus(req, res) {
+    const { id, status } = req.params;
+    const isRead = (status === 'read');
```

Comment:
great work so far. However, this statement means that even though status='ekenkfnnf' it will just set isRead to false and pass!. You should check if the right status has been provided(read or unread)